docs/auto-instrumentation-install.md

```diff
@@ -0,0 +1,282 @@
+- OpentTelemetry leverages Kubernetes to implement auto-instrumentation in such a way that your applications can be
+  instrumented not by updating the application source code but rather by utilizing Kubernetes resources.
+- Supports a subset of instrumentation libraries compared to manual instrumentation
+
+Manual Instrumentation
+
+- Takes more setup effort compared to auto-instrumentation but offers more customizability
+- Requires editing application source code to include pre-built OpenTelemetry instrumentation libraries
+- Supports more instrumentation libraries and customization of the exported telemetry data compared to
+  auto-instrumentation
+
+In particular, auto-instrumentation is useful for applications that use widely popular frameworks and libraries, as
+these frameworks often have pre-built instrumentation capabilities already available.
+
+## Steps for setting up auto-instrumentation
+
+### 1. Deploy the Helm Chart with the Operator enabled
+
+Set `operator.enabled=true` when deploying the chart to enable deploying the operator as well.
+If a cert-manager is not available in the cluster (or other TLS certificate source), then you'll need to deploy it
+using `certmanager.enabled=true`. The cert-manager issues TLS certificates the operator requires. You can use the
+commands below to run these steps.
+
+```
+# Check if cert-manager is already installed, don't deploy a second cert-manager.
+kubectl get pods -l app=cert-manager --all-namespaces
+
+# If cert-manager is not deployed.
+helm install splunk-otel-collector -f ./my_values.yaml --set operator.enabled=true,certmanager.enabled=true -n monitoring helm-charts/splunk-otel-collector
+
+# If cert-manager is already deployed.
+helm install splunk-otel-collector -f ./my_values.yaml --set operator.enabled=true -n monitoring helm-charts/splunk-otel-collector
+```
+
+### 2. Deploy the opentelemetry.io/v1alpha1 Instrumentation
+
+This Instrumentation object is a spec to configure what instrumentation libraries to use for instrumentation. An
+Instrumentation object must be available to the target pod for auto-instrumentation to function. Here is an example
+of a Instrumentation yaml file and how to install it.
+
+<details open>
+<summary>Expand for sample splunk-instrumentation.yaml</summary>
+
+```yaml
+# splunk-instrumentation.yaml
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: splunk-instrumentation
+spec:
+  exporter:
+    endpoint: http://$(SPLUNK_OTEL_AGENT):4317
+  propagators:
+    - tracecontext
+    - baggage
+    - b3
+  java:
+    env:
+      - name: SPLUNK_OTEL_AGENT
+        valueFrom:
+          fieldRef:
+            apiVersion: v1
+            fieldPath: status.hostIP
+      - name: OTEL_EXPORTER_OTLP_ENDPOINT
+        value: https://$(SPLUNK_OTEL_AGENT):4318
+      - name: OTEL_TRACES_EXPORTER
+        value: otlp
+  dotnet:
+    image: your-customized-auto-instrumentation-image:dotnet
```

Comment:
It's confusing to me, as a user, what is `your-customized-auto-instrumentation-image:dotnet`. Why this option needs to be highlighted?

Reply:
I'll remove the other libraries that are not java for now. We just tested dotnet and I think notes like this will be good for developers but not for customers. I'll add some dev notes else where in a follow up PR.